<launch>
    <node pkg="pioneer_control" type="teleop_pioneer_keyboard" name="teleop_keyboard" output="screen">
      <remap from="vel_cmd" to="base_controller/command" />
      <remap from="motor_state_cmd" to="/motor_state_cmd"/>
      <remap from="cmd_vel" to="/cmd_vel"/>
      <remap from="gripper_cmd" to="/gripper_cmd"/>
      <remap from="ptz_cmd" to="/ptz_cmd"/>

      <param name="walk_vel" value="0.05" />
      <param name="run_vel" value="0.5" />
      <param name="yaw_rate" value="0.25" />
      <param name="yaw_run_rate" value="1.5" />
      <param name="pan_rate" value="5" />
      <param name="tilt_rate" value="5" />
      <param name="zoom_rate" value="196" />
      <param name="fast_pan_rate" value="20" />
      <param name="fast_tilt_rate" value="20" />
      <param name="fast_zoom_rate" value="392" />
    </node>
</launch>

